File: java/Workspace/Fishy/src/screens/ScreenManager.java
package screens;

import java.awt.Graphics;

public class ScreenManager 
{
	public static Screen currentScreen;
	
	public ScreenManager(Screen screen)
	{
		currentScreen = screen;
	}
	
	public void Update(long dt)
	{
		currentScreen.Update(dt);
	}
	
	public void Draw(Graphics g)
	{
		currentScreen.Draw(g);
	}
}

File: xna/ShootEmUp/ShootEmUp/ShootEmUp/content/screens/ScreenManager.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using Microsoft.Xna.Framework;
using Microsoft.Xna.Framework.Graphics;

namespace ShootEmUp.content.screens
{
    class ScreenManager
    {
        public static Screen currentScreen, previousScreen;

        public ScreenManager()
        {

        }

        public static void CreateScreen(Screen screen)
        {
            previousScreen = currentScreen;
            currentScreen = screen;
        }

        public void Update(GameTime gameTime)
        {
            currentScreen.Update(gameTime);
        }

        public void Draw(SpriteBatch spriteBatch)
        {
            currentScreen.Draw(spriteBatch);
        }
    }
}
